packages/core/src/config/config.ts

```diff
@@ -702,10 +699,6 @@ export class Config {
     return this.listExtensions;
   }
 
-  getExtensionManagement(): boolean {
```

Comment:
Can we keep this flag until we roll out? This allows us to turn off if there are issues. We can do the cleanup after the rollout IMO. Let me know if I understood this right.